builtin/common/vector.lua

```diff
@@ -19,31 +21,31 @@ function vector.equals(a, b)
 end
 
 function vector.length(v)
-	return math.hypot(v.x, math.hypot(v.y, v.z))
+	return hypot(v.x, hypot(v.y, v.z))
 end
 
 function vector.normalize(v)
 	local len = vector.length(v)
 	if len == 0 then
-		return {x=0, y=0, z=0}
+		return {x = 0, y = 0, z = 0}
 	else
 		return vector.divide(v, len)
 	end
 end
 
 function vector.floor(v)
 	return {
-		x = math.floor(v.x),
-		y = math.floor(v.y),
-		z = math.floor(v.z)
+		x = floor(v.x),
+		y = floor(v.y),
+		z = floor(v.z)
 	}
 end
 
 function vector.round(v)
 	return {
-		x = math.floor(v.x + 0.5),
-		y = math.floor(v.y + 0.5),
-		z = math.floor(v.z + 0.5)
+		x = floor(v.x + 0.5),
+		y = floor(v.y + 0.5),
+		z = floor(v.z + 0.5)
```

Comment:
Maybe you can put this and many other tables in vector.lua in a single line (as long as it doesn't exceed 80 letters).